x/or/da/keeper/cc.go

```diff
@@ -0,0 +1,5 @@
+package keeper
+
+func (k Keeper) appendSequencerBatch() {
```

Comment:
It's better to leave some comment on the lint check failure, as similar check failures may be reported in another PR later on.